lib/matplotlib/widgets.py

```diff
@@ -753,21 +730,12 @@ def __init__(self, ax, label, initial='',
 
         self.capturekeystrokes = False
 
-    @cbook.deprecated("3.4")
-    @property
-    def cnt(self):
-        # Not real, but close enough.
-        return sum(len(d) for d in self._observers.callbacks.values())
-
-    @cbook.deprecated("3.4")
-    @property
-    def change_observers(self):
-        return self._observers.callbacks['change']
-
-    @cbook.deprecated("3.4")
-    @property
-    def submit_observers(self):
-        return self._observers.callbacks['submit']
+    cnt = cbook.deprecated("3.4")(property(  # Not real, but close enough.
```

Comment:
This is a big enough block that it doesn't matter where you put it, but I would put it up there with `params_to_disable`.

But if you don't, you should put `params_to_disable` here, so that it's all in one block.